atc/db/lock/lock.go

```diff
@@ -147,42 +148,44 @@ type LockDB interface {
 type lock struct {
 	id LockID
 
-	logger       lager.Logger
-	db           LockDB
-	locks        lockRepo
-	acquireMutex *sync.Mutex
+	logger lager.Logger
+	db     LockDB
+	locks  lockRepo
+	mutex  *mutex
 
 	acquired LogFunc
 	released LogFunc
 }
 
 func (l *lock) Acquire() (bool, error) {
-	l.acquireMutex.Lock()
-	defer l.acquireMutex.Unlock()
+	if l.mutex.Lock(time.Second) {
```

Comment:
Hey, little nit - wdyt of having the `time.Second` here as a constant, for the purpose of being explicit that that is the timeout we're using? 

thanks!

Reply:
Fixed